.gitlab-ci.yml

```diff
@@ -41,6 +41,7 @@ parthenon-cuda-unit:
     - cd build-cuda-debug
     - cmake -DCMAKE_BUILD_TYPE=Debug
       -DMACHINE_VARIANT=cuda-mpi
+      -DNUM_MPI_PROC_TESTING=1
```

Comment:
Are these needed given the changes to the CI machine file?

Reply:
Hmm